### PR TITLE
Ask user to enable CDN support to render IPyWidget

### PR DESCRIPTION
**Pull request by DonJayamanne**
For #5074

### CODE REVIEW

File: src/datascience-ui/ipywidgets/kernel/index.ts [hunk 1]
```diff
@@ -114,15 +114,6 @@ export function disposeOutput(e: { outputId: string } | undefined) {
         );
     }
 }
-function renderErrorInLastOutputThatHasNotRendered(message: string) {
-    const possiblyEmptyOutputElement = [...stackOfWidgetsRenderStatusByOutputId]
-        .reverse()
-        .find((item) => !item.success);
-    if (possiblyEmptyOutputElement) {
-        //
-        console.log(message);
```
Comment (DonJayamanne) [inline src/datascience-ui/ipywidgets/kernel/index.ts:123]: This is what was missing, I'm now display the message as a messagebox to the user.

File: package.nls.zh-cn.json [hunk 2]
```diff
@@ -433,7 +433,8 @@
     "DataScience.loadClassFailedWithNoInternet": "加载错误 {0}:{1}。需要连接互联网才能加载第三方控件。",
     "DataScience.useCDNForWidgets": "控件需要从第三方网站下载支持文件。点击[这里](https://aka.ms/PVSCIPyWidgets)获取更多信息。",
     "DataScience.loadThirdPartyWidgetScriptsPostEnabled": "改变 \"jupyter.widgetScriptSources\" 的设置后请重新启动内核。",
-    "DataScience.enableCDNForWidgetsSetting": "控件需要从第三方网站下载支持文件。点击<a href='https://command:jupyter.enableLoadingWidgetScriptsFromThirdPartySource'>这里</a>启用此功能，或点击<a href='https://aka.ms/PVSCIPyWidgets'>这里</a>获取更多信息。（加载错误 {0}:{1}）。",
+    "DataScience.enableCDNForWidgetsSettingHtml": "控件需要从第三方网站下载支持文件。点击<a href='https://command:jupyter.enableLoadingWidgetScriptsFromThirdPartySource'>这里</a>启用此功能，或点击<a href='https://aka.ms/PVSCIPyWidgets'>这里</a>获取更多信息。（加载错误 {0}:{1}）。",
```
Comment (rchiodo) [inline package.nls.zh-cn.json:436]: Do you speak Mandarin too? Or did google translate do this?

File: src/client/datascience/ipywidgets/commonMessageCoordinator.ts [hunk 3]
```diff
@@ -127,6 +130,22 @@ export class CommonMessageCoordinator {
...
+                );
+            } else if (!payload.cdnsUsed) {
+                errorMessage = localize.DataScience.enableCDNForWidgetsSetting().format(
+                    payload.moduleName,
+                    payload.moduleVersion
+                );
+            }
+            traceError(`Widget load failure ${errorMessage}`, payload);
+
+            this.appSell.showErrorMessage(errorMessage).then(noop, noop);
```
Comment (rchiodo) [inline src/client/datascience/ipywidgets/commonMessageCoordinator.ts:147]: Seems weird to put up a message in sendLoadFailureTelemetry. Unexpected. Would it be better to split out the error message into a separate function?

**Review by rchiodo — APPROVED** [review]

File: package.nls.zh-cn.json (hunk 2 above)
Comment (DonJayamanne) [reply package.nls.zh-cn.json:436]: I'm a man of many talents

Comment (DonJayamanne) [reply package.nls.zh-cn.json:436]: One of them is `copy paste`, i just copied the previous line. The only different was I removed the HTML tags, thats it.

File: src/client/datascience/ipywidgets/commonMessageCoordinator.ts (hunk 3 above)
Comment (DonJayamanne) [reply src/client/datascience/ipywidgets/commonMessageCoordinator.ts:147]: Renamed the function

File: src/datascience-ui/ipywidgets/kernel/index.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/datascience-ui/ipywidgets/kernel/index.ts:123]: FYI @rchiodo I was getting the prompt in Native Notebooks.
Had to find what was missing, & the only missing part was this area.
Lets see when this gets validated.

File: src/client/datascience/ipywidgets/commonMessageCoordinator.ts [hunk 4]
```diff
@@ -48,6 +49,7 @@ export class CommonMessageCoordinator {
     }
     private ipyWidgetMessageDispatcher?: IIPyWidgetMessageDispatcher;
     private ipyWidgetScriptSource?: IPyWidgetScriptSource;
+    private appSell: IApplicationShell;
```
Comment (IanMatthewHuff) [inline src/client/datascience/ipywidgets/commonMessageCoordinator.ts:52]: appShell. You know I'm always watching 👀

**Review by IanMatthewHuff — APPROVED** [review]
Looks good with the typo fixed.

**Review by joyceerhl — APPROVED** [review]

**Pull request merged**